authenticate/authenticate.go

```diff
@@ -181,6 +184,10 @@ func New(opts config.Options) (*Authenticate, error) {
 		return nil, err
 	}
 
+	administrator := make(map[string]struct{}, len(opts.Administrators))
```

Comment:
Will this be hot reloaded?

Reply:
@desimone I think no, currently, only policy related rules are hot reloaded.

Reply:
https://github.com/pomerium/pomerium/issues/982